roles/config-rh-sso/tasks/realm.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: "Set realm facts for realm: {{ item.name }}"
+  set_fact:
```

Comment:
No need to use `set_fact` here - it seems to just complicates things since you also have to unset below... just reference them directly in the jinja2 template. If you don't want to use `item` in the jinja-template (which may be the reason for doing set_fact?), you can always just set a `loop_var`.

Reply:
Thank you for the suggestion!